apps/workspace-host/src/lib/aws.ts

```diff
@@ -19,7 +19,7 @@ const awsConfigProvider = makeAwsConfigProvider({
           accessKeyId: 'S3RVER',
           secretAccessKey: 'S3RVER',
         },
-        endpoint: 'http://localhost:5000',
+        endpoint: 'http://127.0.0.1:5000',
```

Comment:
This was not necessary in my local tests, but I see how this can be useful to future-proof it in case it ever starts using IPv6. That said, the same endpoint exists in apps/prairielearn/src/lib/aws.ts and in the README for packages/aws.

Reply:
Thanks, changed in 8cb90a4. While this is probably not strictly necessary, it's a "can't hurt, better safe than sorry" change.